openedx/core/djangoapps/catalog/management/commands/cache_programs.py

```diff
@@ -48,33 +143,27 @@ def handle(self, *args, **options):
         except:  # pylint: disable=bare-except
             logger.error('Failed to retrieve program UUIDs.')
```

Comment:
Include the site name in this message and the one above.

If this command fails to retrieve UUIDs for one site, does it make sense to raise an error and give up on the other sites? I don't think it does. We still want to cache programs for edx.org, even if there's a problem with a white label site.